android/build.gradle

```diff
@@ -26,6 +26,6 @@ subprojects {
     project.evaluationDependsOn(':app')
 }
 
-task clean(type: Delete) {
+tasks.register("clean", Delete) {
```

Comment:
Why this was changed?

Reply:
It changed after upgrading gradle version